pdns/dnsdist.cc

```diff
@@ -2258,6 +2336,15 @@ try
 
   auto todo=setupLua(false, g_cmdLine.config);
 
+  if (g_policy.getLocal()->name == "chashed") {
+    vinfolog("Pre-computing hashes for consistent hash load-balancing policy");
```

Comment:
I haven't tested but it looks like this only work for the global load-balancing policy, perhaps we should also test pool-specific load-balancing policies (set with `setPoolServerPolicy()`)?

Reply:
https://github.com/PowerDNS/pdns/pull/6737/files#diff-194369649d922d5a26ab47ac9c895d33R770
Indeed, this only tests against global policy in order to decide whether or not to pre-compute hashes. However, they will be computed on demand if needed.

Reply:
Right, I agree this is not an issue, but I was just wondering if we might want to iterate over the pools there and check their policy as well? But that's not blocking either way.

Reply:
I'll update the check :)